Use correct online help and Git repository according to the correct QGIS Server wrapper used

Gustry opened the pull request:
As now, it's not only Py-QGIS-Server which can be used, it can also be QJazz : 

![image](https://github.com/user-attachments/assets/27ca56c1-9b6f-43b8-9185-06727abeac2f)

CF work started in https://github.com/3liz/qgis-lizmap-server-plugin/pull/100

I also moved the commit ID next to the version number, same as for LWC main version and plugins.

PS : I have changed the name, **temporary**, in source code itself for the screenshot, for trying for now, waiting on https://github.com/3liz/qgis-lizmap-server-plugin/pull/102 and https://github.com/3liz/lizmap-web-client/pull/4991

nworr commented on lizmap/modules/admin/locales/en_US/admin.UTF-8.properties:
```diff
@@ -272,14 +272,14 @@ server.information.qgis.label=QGIS Server
 server.information.qgis.plugin=Plugin
 server.information.qgis.plugin.version=Version
 server.information.qgis.plugin.help=Help
+server.information.not.installed=Not installed
```
```suggestion
server.information.wrapper.not.installed=Not installed
```

nworr commented on lizmap/modules/admin/templates/server_information.tpl:
```diff
@@ -123,32 +125,38 @@
+                </td>
+            {else}
+                {* Generic name for Py-QGIS-Server and QJazz ? *}
+                <th>QGIS Server wrapper</th><td>{@admin.server.information.not.installed@}</td>
```
```suggestion
                <th>QGIS Server wrapper</th><td>{@admin.server.information.wrapper.not.installed@}</td>
```

nworr reviewed (commented):
a small suggestion for a more precise locale name

but, since the wrapper is not mandatory, maybe this line is useless when wrapper is absent ?

Gustry commented:
I have fixed these strings.

> but, since the wrapper is not mandatory, maybe this line is useless when wrapper is absent ?

I have added it so it's make a kind of advertisement for this "product". We were talking to slowly make it recommended
